wire-ios-data-model/Source/Analytics/SegmentationEntry.Conversation.ConversationType+initWithZMConversationType.swift

```diff
@@ -16,29 +16,18 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-/// Represents a key-value pair for analytics event segmentation.
-///
-/// This struct is used to provide additional, structured information about an analytics event.
-/// Each `SegmentationEntry` consists of a key (identifying the type of information) and a value
-/// (the actual data point).
+import WireAnalytics
 
-struct SegmentationEntry: Hashable, Sendable {
```

Comment:
This is actually just renamed, see file `WireAnalytics/Sources/WireAnalytics/Events/Segmentation/AnalyticsEvent+Segmentation.swift`